engine/language-server/src/main/scala/org/enso/languageserver/runtime/ContextEventsListener.scala

```diff
@@ -186,52 +182,21 @@ final class ContextEventsListener(
   private def runExpressionUpdates(
     expressionUpdates: Vector[Api.ExpressionUpdate]
   ): Unit = {
-    def toMethodPointer(call: Api.MethodPointer): (String, String, String) =
-      (call.module, call.definedOnType, call.name)
-
-    val methodPointerToExpression =
-      expressionUpdates.flatMap { update =>
-        update.methodCall.map(call =>
-          toMethodPointer(call) -> update.expressionId
-        )
-      }.toMap
-    val methodPointers = methodPointerToExpression.keys.toSeq
-    repo
-      .getAllMethods(methodPointers)
-      .map { suggestionIds =>
-        val methodPointerToSuggestion =
-          methodPointers
-            .zip(suggestionIds)
-            .collect { case (ptr, Some(suggestionId)) =>
-              ptr -> suggestionId
-            }
-            .toMap
-        val computedExpressions = expressionUpdates.map { update =>
-          ContextRegistryProtocol.ExpressionUpdate(
-            update.expressionId,
-            update.expressionType,
-            update.methodCall.flatMap { call =>
-              val pointer = toMethodPointer(call)
-              methodPointerToSuggestion.get(pointer) match {
-                case suggestionId @ Some(_) => suggestionId
-                case None =>
-                  logger.error("Unable to find suggestion for [{}].", pointer)
-                  None
-              }
-            },
-            update.profilingInfo.map(toProtocolProfilingInfo),
-            update.fromCache,
-            toProtocolPayload(update.payload)
-          )
-        }
-        val payload = ContextRegistryProtocol.ExpressionUpdatesNotification(
-          contextId,
-          computedExpressions
-        )
-        DeliverToJsonController(rpcSession.clientId, payload)
-
-      }
-      .pipeTo(sessionRouter)
+    val computedExpressions = expressionUpdates.map { update =>
```

Comment:
The code is simplified. Instead of trying to find a _suggestion id_ we just send a tripple `(module, type, methodname)`